pkg/smi/health.go

```diff
@@ -9,38 +9,40 @@ import (
 	"github.com/openservicemesh/osm/pkg/version"
 )
 
+var (
+
+	// The key is the API Resource.Kind and the value is the SMI CRD Type
+	reqVersions = map[string]string{
```

Comment:
I realize this isn't really part of this change, but a better name for this may be `reqKinds`.